Docker-files/README.md

```diff
@@ -0,0 +1,78 @@
+# KALDodocker
```

Comment:
Cool name! Let's just call it `Docker files for kALDo`.

Reply:
Will do.